color_code_checker.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
```

Comment:
Good approach to segregate test code and production code. 
File naming can be improved (nothing is wrong here!), color_code_checker instantly does not give a feeling that it contains test code.
Maybe you can think of a slightly more intuitive name.